backend/cellguide/pipeline/constants.py

```diff
@@ -15,4 +15,4 @@
 
 CELL_GUIDE_DATA_BUCKET_PATH_PREFIX = "s3://cellguide-data-public-"
 
-CELL_GUIDE_PINNED_SCHEMA_VERSION = "5.0.0"
+CELL_GUIDE_PINNED_SCHEMA_VERSION = "5.1.0"
```

Comment:
I remember you asking about this previously--after some more thought I think we should leave it at 5.0.0 and let Apps team handle when to bump it. 

We do want them to update as soon as possible but there may be a reason they want to delay